demo/meeting/server.js

```diff
@@ -34,119 +36,132 @@ const log = message => {
-            //   SnsTopicArn: 'Paste your arn here'
-            // }
-          })
-          .promise();
-        attendeeCache[title] = {};
-      }
-      const joinInfo = {
-        JoinInfo: {
-          Title: title,
-          Meeting: meetingCache[title].Meeting,
-          Attendee: (
-            await chime
-              .createAttendee({
-                MeetingId: meetingCache[title].Meeting.MeetingId,
-                ExternalUserId: uuid(),
-              })
-              .promise()
-          ).Attendee,
-        },
-      };
-      attendeeCache[title][joinInfo.JoinInfo.Attendee.AttendeeId] = name;
-      response.statusCode = 201;
-      response.setHeader('Content-Type', 'application/json');
-      response.write(JSON.stringify(joinInfo), 'utf8');
-      response.end();
-      log(JSON.stringify(joinInfo, null, 2));
-    } else if (request.method === 'GET' && request.url.startsWith('/attendee?')) {
-      const query = url.parse(request.url, true).query;
-      const attendeeInfo = {
-        AttendeeInfo: {
-          AttendeeId: query.attendee,
-          Name: attendeeCache[query.title][query.attendee],
-        },
-      };
-      response.statusCode = 200;
-      response.setHeader('Content-Type', 'application/json');
-      response.write(JSON.stringify(attendeeInfo), 'utf8');
-      response.end();
-      log(JSON.stringify(attendeeInfo, null, 2));
-    } else if (request.method === 'POST' && request.url.startsWith('/meeting?')) {
-      const query = url.parse(request.url, true).query;
-      const title = query.title;
-      if (!meetingCache[title]) {
-        meetingCache[title] = await chime
-          .createMeeting({
-            ClientRequestToken: uuid(),
-            // NotificationsConfiguration: {
-            //   SqsQueueArn: 'Paste your arn here',
-            //   SnsTopicArn: 'Paste your arn here'
-            // }
+const server = require(protocol).createServer(
+  options,
+  async (request, response) => {
+    log(`${request.method} ${request.url} BEGIN`);
+    compression({})(request, response, () => {});
+    try {
+      if (
+        request.method === 'GET' &&
+        (request.url === '/' ||
+          request.url === '/v2/' ||
+          request.url.startsWith('/?'))
+      ) {
+        response.statusCode = 200;
+        response.setHeader('Content-Type', 'text/html');
+        response.end(fs.readFileSync(`dist/${app}.html`));
+      } else if (
+        request.method === 'POST' &&
+        request.url.startsWith('/join?')
+      ) {
+        const query = url.parse(request.url, true).query;
+        const title = query.title;
+        const name = query.name;
+        const region = query.region || 'us-east-1';
+
+        if (!meetingCache[title]) {
+          meetingCache[title] = await chime
+            .createMeeting({
+              ClientRequestToken: uuid(),
+              MediaRegion: region
+            })
+            .promise();
+          attendeeCache[title] = {};
```

Comment:
Most of this is auto-formatted changes. This is the only part I've actually changed